TriblerGUI/widgets/trustpage/js/help_page.js

```diff
@@ -0,0 +1,333 @@
+    drawReputationSvg();
+    drawNodeSizeSvg();
+    drawConnectionSizeSvg();
+    drawDividingLineSvg();
+}
+
+function drawIdentitySvg () {
+    var svgIdentity = d3.select("#help-circle-text");
+
+    var groups = svgIdentity.append("svg")
+        .attr("class", "node")
+        .attr("overflow", "visible")
+        .attr("x", parseInt(svgIdentity.style("width"), 10) / 2)
+        .attr("y", parseInt(svgIdentity.style("height"), 10) / 2);
+
+    groups
+        .append("circle")
+        .attr("r", config.node.circle.maxRadius)
+        .attr("fill", config.triblerOrange);
+
+    groups
+        .append("text")
+        .attr("dominant-baseline", "central")
+        .attr("text-anchor", "middle")
+        .style("font-family", config.node.publicKeyLabel.fontFamily)
+        .style("font-size", 12)
+        .style("font-weight", config.node.publicKeyLabel.fontWeight)
+        .style("fill", config.node.publicKeyLabel.color)
+        .text("#ABC")
+}
+
+function drawReputationSvg () {
+    var svgNodeColor = d3.select("#help-circle-color");
+
+    var groups = svgNodeColor.append("svg")
+        .attr("id", "reputation-legend")
+        .attr("x", 0)
+        .attr("y", parseInt(svgNodeColor.style("height"), 10) / 2)
+        .attr("overflow", "visible");
+
+    var nodeRadius = config.help.circleColor.nodeRadius;
+    var names = config.help.circleColor.colorNames;
+    var numberOfNodes = config.help.numberOfNodes;
+    var labelOffset = config.help.labelOffset;
+    var width = parseInt(svgNodeColor.style("width"), 10);
+    var spacing = width / (numberOfNodes + 1);
+    var nodesData = [];
+
+    for(var i = 0; i < numberOfNodes; i++) {
+        nodesData.push({
+            x: (i + 1) * spacing,
+            name: names[i],
+            color: nodeColor(i / (numberOfNodes - 1))
+        })
+    }
+
+    groups.selectAll("circle")
+        .data(nodesData)
+        .enter()
+        .append("circle")
+        .attr("r", nodeRadius)
+        .attr("fill", function (node) {
+            return node.color;
+        })
+        .attr("cx", function (node) {
+            return node.x;
+        });
+
+    var reputationLabel = document.getElementsByTagName('text')[0];
+    var labelHeight = reputationLabel.getBoundingClientRect().height;
+
+    groups.selectAll("text")
+        .data(nodesData)
+        .enter()
+        .append("text")
+        .attr("dominant-baseline", "central")
+        .attr("text-anchor", "middle")
+        .style("font-family", config.node.publicKeyLabel.fontFamily)
+        .style("font-size", 12)
+        .style("font-weight", config.node.publicKeyLabel.fontWeight)
+        .style("fill", "#FFFFFF")
+        .attr("x", function (node) {
+            return node.x;
+        })
+        .attr("y", (1/2) * labelHeight + nodeRadius + labelOffset)
+        .text(function (node) {
+            return node.name;
+        });
+}
+
+function drawNodeSizeSvg () {
+    var svgNodeSize = d3.select("#help-circle-size");
```

Comment:
You can use `var` only once and then the different variables separated with comma's.

`var x = 5,
       y = 6;`